src/main/java/org/folio/service/PasswordRestLinkService.java

```diff
@@ -0,0 +1,18 @@
+package org.folio.service;
+
+import io.vertx.core.Future;
+import org.folio.rest.util.OkapiConnectionParams;
+
+/**
+ * Service for password reset link functionality
+ */
+public interface PasswordRestLinkService {
```

Comment:
seems like the name should be PasswordResetLinkService

Reply:
Done